flatgeobuf/README.md

```diff
@@ -0,0 +1,44 @@
+# FlatGeobuf Encoding Specification
+
+The FlatGeobuf encoding defines how field boundaries compliant to fiboa must be published.
+The generic FlatGeobuf format is defined on the [FlatGeobuf website](https://flatgeobuf.org/).
+We aim to support any future version of FlatGeobuf, too.
+
+> [!NOTE]
+> The FlatGeobuf encoding is still work in progress. Feedback is welcome!
+
+- **[Examples](examples/)**
+- **[Data type mapping](datatypes.md)**
+
+## Collection
+
+The FlatGeobuf file must provide the [fiboa Collection](../core/README.md#collection) as a separate JSON file.
+Optionally, if the tool supports it, the fiboa collection can be written into the `Header` table's 
+[`metadata`](https://github.com/flatgeobuf/flatgeobuf/blob/ee7c8f5f45c67dd4a84a51fef518dfebc3e19d0a/src/fbs/header.fbs#L81) 
+field in a property named `fiboa` within a JSON-structured object.
+
+> [!NOTE]
+> As of GDAL/OGR 3.9.0 the [FlatGeobuf driver](https://gdal.org/drivers/vector/flatgeobuf.html)
+does not support writing the Header's `metadata` field. 
```

Comment:
```suggestion
> does not support writing the Header's `metadata` field. 
```

Reply:
Yes, you are right. It wasn't obvious to me from the [driver's documentation page](https://gdal.org/drivers/vector/flatgeobuf.html), but confirmed in https://github.com/OSGeo/gdal/issues/10183#issuecomment-2158690715 that `-mo` is supported now.